src/Adapter/PlatformServices.Shared/netstandard1.0/Constants.cs

```diff
@@ -3,13 +3,20 @@
 
 namespace Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices
 {
+    using System.Collections.Generic;
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+
 #pragma warning disable SA1649 // SA1649FileNameMustMatchTypeName
 
     internal class Constants
     {
+        public static readonly TestProperty DeploymentItemsProperty = TestProperty.Register("MSTestDiscoverer.DeploymentItems", DeploymentItemsLabel, typeof(KeyValuePair<string, string>[]), TestPropertyAttributes.Hidden, typeof(TestCase));
```

Comment:
>Property [](start = 59, length = 8)

In shared folder, I am seeing there are few files in netstandard1.0 and netstandard1.3. Can you please tell how we decide which file to go in which folder?

Reply:
When the files use libraries of ns1.5 we put it in ns1.3 folder. Folder ns1.3 was created when it was the latest and ns1.5 was not out.
When the files use the libraries of ns 1.0 only, we put it in ns 1.0 folder.

This has been done to make the movement of files in specific target framework projects in future if needed.